client/src/components/Button/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from 'components/Button/Button.module.scss';
+
+const Button = ({ text, disabled }) => {
+  return (
+    <div className={styles.authBtnDiv}>
+      <button type="submit" disabled={disabled}>
+        {text}
+      </button>
+    </div>
+  );
```

Comment:
you dont need the return keyword

`const Button = ({ text, disabled }) => (
    <div className={styles.authBtnDiv}>
      <button type="submit" disabled={disabled}>
        {text}
      </button>
    </div>
);`